assets/product-form.js

```diff
@@ -23,11 +23,12 @@ if (!customElements.get('product-form')) {
 
       const config = fetchConfig('javascript');
       config.headers['X-Requested-With'] = 'XMLHttpRequest';
-      config.body = JSON.stringify({
-        ...JSON.parse(serializeForm(this.form)),
-        sections: this.cartNotification.getSectionsToRender().map((section) => section.id),
-        sections_url: window.location.pathname
-      });
+      delete config.headers['Content-Type'];
```

Comment:
Oh, it's because we're no longer sending a `Content-Type` `application/json` as part of the header.

The function that returns these config headers is still being used in other endpoints across the theme, and they're still sending JSON requests, so that's why I only removed this from the `/cart` request.

Reply:
Ah great. Thanks for the explanation 🙂